packages-tests/NodeCollector/BinaryOpTreeRootLocatorTest.php

```diff
@@ -49,51 +49,51 @@ public function testRightAssociative(): void
         $a->setAttribute(AttributeKey::PARENT_NODE, $tree);
         $bc->setAttribute(AttributeKey::PARENT_NODE, $tree);
 
-        $this->assertSame($tree, $locator->findOperationRoot($a, Plus::class));
-        $this->assertSame($bc, $locator->findOperationRoot($b, Plus::class));
-        $this->assertSame($bc, $locator->findOperationRoot($c, Plus::class));
-        $this->assertSame($bc, $locator->findOperationRoot($bc, Plus::class));
+        $this->assertSame($tree, $binaryOpTreeRootLocator->findOperationRoot($a, Plus::class));
+        $this->assertSame($bc, $binaryOpTreeRootLocator->findOperationRoot($b, Plus::class));
+        $this->assertSame($bc, $binaryOpTreeRootLocator->findOperationRoot($c, Plus::class));
+        $this->assertSame($bc, $binaryOpTreeRootLocator->findOperationRoot($bc, Plus::class));
     }
 
     public function testWrongRootOp(): void
     {
-        $locator = new BinaryOpTreeRootLocator();
+        $binaryOpTreeRootLocator = new BinaryOpTreeRootLocator();
 
         // (Minus (Plus a b) c)
         $a = new Variable('a');
         $b = new Variable('b');
-        $ab = new Plus($a, $b);
-        $a->setAttribute(AttributeKey::PARENT_NODE, $ab);
-        $b->setAttribute(AttributeKey::PARENT_NODE, $ab);
+        $plus = new Plus($a, $b);
```

Comment:
IDK, this does not look like an improvement to me. Previously, the var name reflected the structure and was consistent with the other test.

Reply:
`$ab` can be anything.
The rule takes into account the type that has to be suffix of the variable name, so it's clear further in text what object is it.

Correct naming here might be `$abPlus`